torch/csrc/autograd/VariableTypeUtils.h

```diff
@@ -47,6 +47,11 @@ inline void check_inplace(const Tensor& tensor) {
       auto diff_view_meta = static_cast<DifferentiableViewMeta*>(impl::get_autograd_meta(var));
       // This can throw or warn
       handle_view_on_rebase(diff_view_meta);
+      auto base = tensor._base();
```

Comment:
nit: no need to create a temporary Tensor here.
And if you do so, you can take a const reference to avoid the refcount bumps.